rasa/core/channels/slack.py

```diff
@@ -506,6 +512,11 @@ async def webhook(request: Request) -> HTTPResponse:
                 user_message = event.get("text", "")
                 sender_id = event.get("user", "")
                 metadata = self.get_metadata(request)
+                channel_id = metadata.get("out_channel")
+                thread_id = metadata.get("thread_id")
+                conversation_id = self._get_conversation_id(
+                    sender_id, channel_id, thread_id
```

Comment:
Think to avoid confusion, it might be good to replace `sender_id` with `user_id` and `conversation_id` with `sender_id`. Wdyt?

Reply:
i wanted to make sure people understand we are trying to compute a `conversation_id` instead of `sender_id`. I can replace `sender_id` if you feel that would be better.

Reply:
I see what you mean. We use this below when we call `process_message` and we currently do 
```
sender_id=conversation_id,
```
I think it makes sense to rename the variables as you recommend. `sender_id` is Rasa's own name, whereas `user_id` we get from Slack (along with channel and thread IDs).

Reply:
yeah, that is a good point -- I agree that conversation_id is a much nicer name -- the confusion would just be caused on our side because of that historic name 😅  . Would be great if we could replace that `sender_id` by `user` or `user_id` or so 👍